src/components/modals/video/index.jsx

```diff
@@ -1,5 +1,7 @@
-import { useEffect } from "react";
+import React, { useEffect } from "react";
+
 import { AiFillCloseCircle } from "react-icons/ai";
+import { attachClasses } from "../../../utils/index";
```

Comment:
1. This import statement should be after the classes import
2.  No need to add index in ../../../utils/index - only ../../../utils will work fine